src/Uno.UI.Tests/App/Behaviors/AttachedPropertiesBehavior.cs

```diff
@@ -19,7 +19,7 @@ public static class AttachedPropertiesBehavior
 				"CustomText",
 				typeof(string),
 				typeof(AttachedPropertiesBehavior),
-				new FrameworkPropertyMetadata(string.Empty, OnCustomTextChanged));
+				new PropertyMetadata(string.Empty, OnCustomTextChanged));
```

Comment:
Why ?

Reply:
`FrameworkPropertyMetadata` is internal to Uno - it doesn't compile on UWP.